csr_check.go

```diff
@@ -22,8 +23,8 @@ const (
 	nodeGroup      = "system:nodes"
 	nodeUserPrefix = nodeUser + ":"
 
-	maxPendingDelta = time.Hour
-	maxPendingCSRs  = 100
+	maxDeniedDelta = time.Hour
```

Comment:
How about increasing `maxPendingDelta` only for the the node client and serving certs? This way we wont deny any CSRs unwantedly.